### DISCUSSION TITLE
Benutzermanagement

### DISCUSSION THREAD

**Discussion by kevinhaybach** [Zefau/ioBroker.jarvis#1967] [Ideas / Feature Requests]
Hallo zusammen,

entschuldigt wenn es diese Idee bereits gab aber ich habe Sie nirgends gefunden, auch wenn ich es mir nicht vorstellen kann. Besteht die Möglichkeit verschiedene Benutzer mit verschiedenen Berechtigungen einzufügen? Z.B. sollen meine Gäste über die Visualisierung das Licht steuern können aber nicht die Haustür öffnen.
Hierzu wären aber verschiedene Benutzer die dann über Passwörter geschützt sind notwendig.

Mit freundlichen Grüßen
Kevin

**Comment by kevinhaybach** [comment]
Ich habe gesehen, dass es in der Pro Version die Möglichkeit gibt ein Passwort zu vergeben um die Seiten zu öffnen und um Jarvis zu konfigurieren. Wenn es jetzt noch die Möglichkeit gäbe einzelne Tabs mit einem Passwort zu schützen wäre ich rundum zufrieden :)

**Comment by Zefau** [comment]
dann bitte für https://github.com/Zefau/ioBroker.jarvis/issues/965 voten
